cli/src/utils/renderVersionWarning.ts

```diff
@@ -47,7 +52,7 @@ function checkForLatestVersion(): Promise<string> {
             let body = "";
             res.on("data", (data) => (body += data));
             res.on("end", () => {
-              resolve(JSON.parse(body).latest);
+              resolve((JSON.parse(body) as DistTagsBody).latest);
```

Comment:
Could use Zod here also?

Reply:
That would make sense! Basically anywhere that `JSON.parse`s 😄